docs/orleans/host/configuration-guide/activation-collection.md

```diff
@@ -1,20 +1,30 @@
 ---
-title: Activation garbage collection
-description: Learn about activation garbage collection in .NET Orleans.
-ms.date: 08/25/2023
+title: Activation collection
+description: Learn about activation collection in .NET Orleans.
+ms.date: 02/07/2024
+zone_pivot_groups: orleans-version
 ---
 
-# Activation garbage collection
+# Activation collection
 
+<!-- markdownlint-disable MD044 -->
+:::zone target="docs" pivot="orleans-7-0"
+<!-- markdownlint-enable MD044 -->
+This article applies to: ✔️ Orleans 7.x and later versions
+:::zone-end
+<!-- markdownlint-disable MD044 -->
+:::zone target="docs" pivot="orleans-3-x"
+<!-- markdownlint-enable MD044 -->
 This article applies to: ✔️ Orleans 3.x and earlier versions
+:::zone-end
 
 A *grain activation* is an in-memory instance of a grain class that gets automatically created by the Orleans runtime on an as-needed basis as a temporary physical embodiment of a grain.
 
-Activation Garbage Collection (Activation GC) is the process of removal from memory of unused grain activations. It's conceptually similar to how garbage collection of memory works in .NET. However, Activation GC only takes into consideration how long a particular grain activation has been idle. Memory usage isn't used as a factor.
+Activation Collection is the process of removal from memory of unused grain activations. It's conceptually similar to how garbage collection of memory works in .NET. However, Activation Collection only takes into consideration how long a particular grain activation has been idle. Memory usage isn't used as a factor.
 
-## How activation GC works
+## How activation collection works
```

Comment:
Activation collection is lower case here but upper case in other places. We should be consistent - I'd suggest lower case.